hw2/04-background.html

```diff
@@ -11,18 +11,21 @@
   </head>
   <body>
     <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
+      <div class="container bg-light w-50 mt-5 p-3 rounded">
+        <h1 class="text-center fs-1 fw-bold">Exercise 04 - Background</h1>
+        <p class="mt-3 mb-3">
+          The background color will change every few seconds (as indicated below).
+          You can stop this behavior and you can also change how often the
+          background color changes.
+        </p>
+        <div class="mt-3 mb-3">
+          <label for="interval">Background change interval (seconds): </label>
+          <input type="number" id="interval"/>
+        </div>
+        <div class="mt-3">
+          <button class="btn btn-primary" id="start" onclick="startColorChange()">Start</button>
+          <button class="btn btn-danger" id="stop" onclick="stopColorChange()" hidden>Stop</button>
```

Comment:
There should not be any in-line JavaScript in the HTML file. All JS code should be in a separate .js file.